docs/configuration.rst

```diff
@@ -832,7 +832,7 @@ Sampling
      
      version_added:
         v0.33.0:
-        v2.15.0: Only applied when DD_TRACE_SAMPLE_RATE, DD_TRACE_SAMPLING_RULES, or DD_SPAN_SAMPLING_RULE are set.
+        v2.15.0: Only applied when DD_TRACE_SAMPLING_RULES, or DD_SPAN_SAMPLING_RULE are set.
```

Comment:
Shouldn't we keep the original v2.15.0 entry and add a new one for v3.0 saying that DD_TRACE_SAMPLE_RATE was removed?

Reply:
Yes! Good catch. This was a victim of find and replace